README.md

```diff
@@ -108,15 +108,15 @@ _You can enable the following settings in Xcode by running [this script](resourc
     public func executeRequest(
       _ request: URLRequest,
       onSuccess: @escaping (ModelType, Bool) -> Void,
-      onFailure: @escaping (Error) -> Void) -> URLSessionCancellable
+      onFailure: @escaping (Error) -> Void) -> URLSessionCancellable)
```

Comment:
```suggestion
      onFailure: @escaping (Error) -> Void)
      -> URLSessionCancellable
```

So actually I think we want this. `URLSessionCancellable` is the return type of the `executeRequest(...)` function. BUT I'm super glad you flagged this, because here we aren't following [this](https://github.com/airbnb/swift#long-function-declaration) guideline, which may have led to the confusion.

Reply:
Yes, you're right. It won't even compile now :P

Reply:
Could you put up another PR? Sorry for the hassle.

Reply:
Sure!